lib/web_console/templates/console.js.erb

```diff
@@ -80,13 +80,34 @@ REPLConsole.prototype.commandHandle = function(line, callback) {
     return status >= 200 && status < 300 || status === 304;
   }
 
+  function parseJSON(text) {
+    try {
+      return JSON.parse(text);
+    } catch (e) {
+      return null;
+    }
+  }
+
+  function getErrorText(xhr) {
+    if (!xhr.status) {
+      return "Oops! Failed to connect to Web Console middleware.\n" +
```

Comment:
It may be my native language parasite skills kicking in, but maybe we can reword the following sections:

```patch
- connect to Web Console
+ connect to the Web Console
```

... and:

```patch
- Please make sure that web server is running.
+ Please make sure that rails development server is running.
```